LICENSE

```diff
@@ -1,34 +1,202 @@
-Copyright (c) 2011 Evolving Web Inc.
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
```

Comment:
Apache-2.0, which the latter contributions through Puppet Labs, Inc., were licensed is more restrictive than the original MIT, it makes sense to have it in the main file. Attributions are still preserved in the NOTICE file, which is recommended by the Apache 2.0 application guidelines.